cmd/registry/cmd/check/lint/checker.go

```diff
@@ -28,6 +28,12 @@ import (
 	"github.com/apigee/registry/server/registry/names"
 )
 
+type contextKey int
+
+const (
+	ContextKeyClient contextKey = iota
```

Comment:
We might want to say `ContextKeyRegistryClient` here -- I know it's more verbose :( but it would align better with `ContextKeyAdminClient` if we decide we need that and it would make the purpose of this more obvious -- it wasn't until I saw the usage below that I realized what "Client" was referring to here :)

Reply:
ack